api/pkg/api/stack_group.go

```diff
@@ -2,26 +2,31 @@ package api
 
 import (
 	"github.com/chanzuckerberg/happy/api/pkg/cmd"
+	"github.com/chanzuckerberg/happy/api/pkg/request"
 	"github.com/chanzuckerberg/happy/api/pkg/response"
 	"github.com/chanzuckerberg/happy/shared/model"
 	"github.com/gofiber/fiber/v2"
 )
 
 type StackHandler struct {
-	stack cmd.Stack
+	stack cmd.StackIface
 }
 
-func MakeStackHandler(s cmd.Stack) *StackHandler {
+func MakeStackHandler(s cmd.StackIface) *StackHandler {
 	return &StackHandler{
 		stack: s,
 	}
 }
 
+func MakeStackHandler2() *StackHandler {
```

Comment:
I think this might have been leftover. Can you remove it?

Reply:
Good catch, removed